ext/standard/proc_open.c

```diff
@@ -1182,6 +1231,34 @@ PHP_FUNCTION(proc_open)
 	childHandle = pi.hProcess;
 	child       = pi.dwProcessId;
 	CloseHandle(pi.hThread);
+#elif defined(USE_POSIX_SPAWN)
+	posix_spawn_file_actions_t factions;
+	int r;
+	posix_spawn_file_actions_init(&factions);
+
+	if (close_parentends_of_pipes(&factions, descriptors, ndesc) == FAILURE) {
+		posix_spawn_file_actions_destroy(&factions);
+		close_all_descriptors(descriptors, ndesc);
+		goto exit_fail;
+	}
+
+	if (cwd) {
+		php_ignore_value(posix_spawn_file_actions_addchdir_np(&factions, cwd));
+	}	
+
+	if(argv) {
```

Comment:
CS: space after if

Reply:
Do we have some clanf-format template to avoid this? ok. will correct that.

Reply:
no we don't

Reply:
the only thing we have are just those guidelines: https://github.com/php/php-src/blob/master/CODING_STANDARDS.md

Reply:
#7933